src/core/Notification/InlineNotification/InlineNotification.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { InlineNotification } from './InlineNotification';
+import { axeTest } from '../../../utils/test';
+
+describe('children', () => {
+  const notificationWithElementChild = (
+    <InlineNotification>
+      <div>Test notification</div>
+    </InlineNotification>
+  );
+  it('has the given content', () => {
+    const { container } = render(notificationWithElementChild);
+    expect(container.firstChild).toHaveTextContent('Test notification');
+  });
+
+  it('should match snapshot', () => {
+    const { container } = render(notificationWithElementChild);
+    expect(container).toMatchSnapshot();
+  });
+});
+
+describe('props', () => {
+  const InlineInlineNotificationComponent = (
+    <InlineNotification labelText="Inline Notification label">
+      Testcontent
+    </InlineNotification>
+  );
+  it('Inline component should contain given labelText', () => {
+    const { getByText } = render(InlineInlineNotificationComponent);
+    const label = getByText('Inline Notification label');
+    expect(label).toHaveClass('fi-notification_label');
+  });
```

Comment:
These seems to test `labelText` prop, so it would be nice to be wrapped in it's own describe block located inside this props block to be consistent.

Reply:
Check also for the Alert.test.tsx for same kind.